### DISCUSSION TITLE
Bering Sea spatial products

### DISCUSSION THREAD

**Discussion by Jason-Conner-NOAA** [afsc-gap-products/model-based-indices#43] [Team Posts]
I have updated the spatial data files used for the Bering Sea:
https://github.com/afsc-gap-products/bering-sea-spatial

We are standing by until @James-Thorson-NOAA is able to include these as the default extrapolation grids in VAST for EBS and NBS, after which, we should be ready to freeze VAST versions for 2022.

**Comment by James-Thorson-NOAA** [comment]
I've added FishStatsUtils release 2.11.0 and VAST release 3.9.0 (where installing VAST 3.9.0 automatically updates to FishStatsUtils >= 2.11.0).  It passes all the checks that I expected on my end (the dev branch contains some further fixes and additional tests).

@Jason-Conner-NOAA could you please test some BS product using VAST 3.9.0 to to a further sniff-test (confirm that the scale makes sense etc.)?

**Comment by Jason-Conner-NOAA** [comment]
Wow, that was fast! Thanks Jim, sure thing - if the VM stays running, should have results on Monday.
